docs/source-lit/conf.py

```diff
@@ -398,20 +398,20 @@ def find_source():
 from typing import Optional
 
 import torch
-import pytorch_lightning as pl
+import lightning.pytorch as pl
 from torch import nn
 from torch.utils.data import IterableDataset, DataLoader, Dataset
-from pytorch_lightning import LightningDataModule, LightningModule, Trainer, seed_everything
-from pytorch_lightning.callbacks import Callback
-from pytorch_lightning.cli import _JSONARGPARSE_SIGNATURES_AVAILABLE as _JSONARGPARSE_AVAILABLE
-from pytorch_lightning.utilities import (
+from pytorch.lightning import LightningDataModule, LightningModule, Trainer, seed_everything
+from pytorch.lightning.callbacks import Callback
+from pytorch.lightning.cli import _JSONARGPARSE_SIGNATURES_AVAILABLE as _JSONARGPARSE_AVAILABLE
+from pytorch.lightning.utilities import (
     _APEX_AVAILABLE,
     _TORCHVISION_AVAILABLE,
     _TORCH_GREATER_EQUAL_1_10,
 )
-from pytorch_lightning.loggers.neptune import _NEPTUNE_AVAILABLE
-from pytorch_lightning.loggers.comet import _COMET_AVAILABLE
-from pytorch_lightning.loggers.mlflow import _MLFLOW_AVAILABLE
-from pytorch_lightning.loggers.wandb import _WANDB_AVAILABLE
+from pytorch.lightning.loggers.neptune import _NEPTUNE_AVAILABLE
+from pytorch.lightning.loggers.comet import _COMET_AVAILABLE
+from pytorch.lightning.loggers.mlflow import _MLFLOW_AVAILABLE
+from pytorch.lightning.loggers.wandb import _WANDB_AVAILABLE
```

Comment:
This is in reverse. Can we change it to lightning.pytorch?

Reply:
great catCT!!